guides/source/active_record_basics.md

```diff
@@ -130,9 +130,9 @@ Active Record uses conventions for column names in the database tables as well,
 depending on the purpose of these columns.
 
 * **Primary keys** - By default, Active Record will use an integer column named
-  `id` as the table's primary key (`bigint` for PostgreSQL and MySQL, `integer`
-  for SQLite). When using [Active Record Migrations](#migrations) to create your
-  tables, this column will be automatically created.
+  `id` as the table's primary key (`bigint` for PostgreSQL, MySQL and MariaDB,
```

Comment:
```suggestion
  `id` as the table's primary key (`bigint` for PostgreSQL, MySQL, and MariaDB,
```

Sorry, I missed this one the first time 😓

Reply:
Updated. Thank you!